build.sbt

```diff
@@ -1,3 +1,5 @@
+scalaVersion in ThisBuild := "2.12.4"
```

Comment:
We want to continue supporting Scala 2.11, don't set the Scala version in the build files.

Reply:
ok